.clusterfuzzlite/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM gcr.io/oss-fuzz-base/base-builder-go
```

Comment:
For CFLite, the Dockerfile is a bit different from OSS-Fuzz:

1. Rather than cloning, we copy in the source code.
2. The Docker build is relative to the project source root (to facilitate 1)

Please see https://google.github.io/clusterfuzzlite/build-integration/#dockerfile for an example. 

This should look something like (but please test it):

```docker
FROM gcr.io/oss-fuzz-base/base-builder-go
COPY . $SRC/sigstore
COPY ./.clusterfuzzlite/build.sh $SRC/ 
WORKDIR $SRC/sigstore/test/fuzz
```

Reply:
Thank you @oliverchang! Appreciate the help!